docs/docs/Concepts/mcp-server.md

```diff
@@ -228,80 +231,7 @@ The default address is `http://localhost:6274`.
-By default, Langflow isn't exposed to the public internet.
-However, you can forward Langflow server traffic with a forwarding platform like [ngrok](https://ngrok.com/docs/getting-started/) or [zrok](https://docs.zrok.io/docs/getting-started).
-
-The following procedure uses ngrok, but you can use any similar reverse proxy or forwarding platform.
-This procedure also assumes that you're using the default Langflow listening address `http://localhost:7860` (`http://localhost:7868` if using Langflow for Desktop).
-
-1. Sign up for an [ngrok account](https://dashboard.ngrok.com/signup).
-
-2. [Install ngrok](https://ngrok.com/docs/getting-started/#1-install-ngrok).
-
-3. Copy your [ngrok authtoken](https://dashboard.ngrok.com/get-started/your-authtoken) and use it to authenticate your local ngrok server:
-
-    ```bash
-    ngrok config add-authtoken NGROK_TOKEN
-    ```
-
-    Replace `NGROK_TOKEN` with your ngrok authtoken.
-
-4. Use ngrok to expose your Langflow server to the public internet:
-
-    ```bash
-    ngrok http http://localhost:7860
-    ```
-
-    The ngrok session starts in your terminal and deploys an ephemeral domain with no authentication.
-    To add authentication or deploy a static domain, see the [ngrok documentation](https://ngrok.com/docs/).
-
-
-    The `Forwarding` row displays the forwarding address for your Langflow server:
-
-    ```
-    Forwarding https://94b1-76-64-171-14.ngrok-free.app -> http://localhost:7860
-    ```
-
-    The forwarding address is acting as a reverse proxy for your Langflow server.
-
-5. From the Langflow dashboard, select the project that contains the flows you want to serve as tools, and then click the **MCP Server** tab.
-
-      Note that the code template now contains your ngrok forwarding address instead of the localhost address:
-
-      ```json
-      {
-        "mcpServers": {
-          "PROJECT_NAME": {
-            "command": "uvx",
-            "args": [
-              "mcp-proxy",
-              "https://94b1-73-64-171-14.ngrok-free.app/api/v1/mcp/project/fdbc12af-0dd4-43dc-b9ce-c324d1ce5cd1/sse"
-            ]
-          }
-        }
-      }
-    ```
-
-6. Complete the steps in [Connect clients to Langflow's MCP server](#connect-clients-to-use-the-servers-actions) using the ngrok forwarding address.
-
-Your MCP client is now connected to your project's MCP server over the public internet.
-
-If using Cursor, your conversations are the same as they are on your local host:
-
-```
-{
-  "input_value": "What job experience does Emily have?"
-}
-Result:
-What job experience does Emily have?
-Emily J. Wilson has the following job experience:
-```
-
-You can use the ngrok console output to monitor requests for your project's endpoint:
-
-```
-16:35:48.566 EDT GET /api/v1/mcp/project/fdbc12af-0dd4-43dc-b9ce-c324d1ce5cd1 200 OK
-```
+To deploy your MCP server externally with ngrok, see [Deploy your Langflow server using ngrok](/deployment-public-server#share-your-mcp-server-externally).
```

Comment:
Can you make this section a subsection of "Connect clients to Langflow's MCP server" or a subsection of "Prerequisites"?

I know we need to keep it bc it's linked from the UI, but it's out of sequence being at the end of the page while also mentioned in the prereqs and connect clients sections.